scrapper.py

```diff
@@ -56,11 +102,11 @@ def get_data(soup: BeautifulSoup, data: dict) -> int:
 
     URL = 'https://krisha.kz'
     # find all offers on page
-    text_data = soup.find('section',
+    list_with_data = soup.find('section',
                             class_='a-list a-search-list a-list-with-favs').findAll('div',
                                                          class_=re.compile('^a-card a-storage-live ddl_product'))
 
-    for each in text_data:
+    for each in list_with_data:
         # parse price of flat
         text_with_one_flat_price = ''.join(each.find('div',
```

Comment:
@ProvVlad m why do you use "".join() function? is it a "pythonic" style of casting to String type? It would be great if you share a link to some doc that explains this.